dom.bs

```diff
@@ -1794,6 +1798,16 @@ interface AbortSignal : EventTarget {
 <p>An {{AbortSignal}} object has an associated <dfn export for=AbortSignal>aborted flag</dfn>. It is
 unset unless specified otherwise.
 
+<p>The static <dfn method for=AbortSignal><code>abort()</code></dfn> steps are:
```

Comment:
This should move a little further down to be just before the definition of `aborted`, but I can also fix this once we have agreement on adding this.